native/src/components/Feedback.tsx

```diff
@@ -109,87 +53,56 @@ const Feedback = ({
   setSearchTerm,
 }: FeedbackProps): ReactElement => {
   const { t } = useTranslation('feedback')
-  const theme = useTheme()
   const navigation = useNavigate().navigation
 
+  const isSearchFeedback = searchTerm !== undefined
   const submitDisabled = isPositiveFeedback === null && comment.trim().length === 0 && !isSearchFeedback
 
-  const renderBox = (): React.ReactNode => {
-    if (['idle', 'failed'].includes(sendingStatus)) {
-      return (
-        <KeyboardAwareScrollView>
-          {isSearchFeedback ? (
-            <>
-              <HeadlineContainer>
-                <Description>{t('searchTermDescription')}</Description>
-              </HeadlineContainer>
-              <Input value={searchTerm} onChangeText={setSearchTerm} />
-            </>
-          ) : (
-            <>
-              <Caption title={t('headline')} />
-              <DescriptionContainer>
-                <Text>{t('description')}</Text>
-              </DescriptionContainer>
-              <FeedbackButtons isPositiveFeedback={isPositiveFeedback} setIsPositiveFeedback={setIsPositiveFeedback} />
-            </>
-          )}
+  if (sendingStatus === 'sending') {
+    return <ActivityIndicator size='large' color='#0000ff' />
```

Comment:
Why did you choose this color? Why not use the Loading component?

Reply:
This is the previous implementation I just didn't change.